platform/ios/src/MGLMapView.mm

```diff
@@ -1899,9 +1899,13 @@ - (void)resetNorthAnimated:(BOOL)animated
 
 - (void)resetPosition
 {
-    MGLMapCamera *camera = [MGLMapCamera camera];
-    camera.altitude = MGLAltitudeForZoomLevel(0, 0, 0, self.frame.size);
-    self.camera = camera;
+    CGFloat pitch = _mbglMap->getDefaultPitch();
+    CLLocationDirection heading = mbgl::util::wrap(_mbglMap->getDefaultBearing(), 0., 360.);
```

Comment:
Correct, we can’t assume the style was authored in Studio. Alternatively, we could establish a convention that `mbgl::Map` wraps the return values of `getDefeaultBearing()` and `getBearing()`, but we can tackle that separately from this PR.